editor/src/clj/editor/collision_object.clj

```diff
@@ -72,32 +76,45 @@
 
 (g/deftype PhysicsType (s/enum "2D" "3D"))
 
+(defn- unique-id-error [node-id id id-counts]
+  (or (validation/prop-error :fatal node-id :id validation/prop-empty? id "Id")
+      (validation/prop-error :fatal node-id :id (partial validation/prop-id-duplicate? id-counts) id)))
```

Comment:
Since we allow empty ids, you only need to keep the second expression from this.